graphs/possible_bipartition.py

```diff
@@ -8,5 +8,37 @@ def possible_bipartition(dislikes):
         Time Complexity: ?
         Space Complexity: ?
     """
-    pass
+    # Create an empty queue
+    dog_queue = deque()
+    kennel_a = []
+    kennel_b = []
+
+    # Add dogs to the queue
+    for i in range(len(dislikes)):
+        dislikes[i].append(i)
+        dog_queue.append(dislikes[i])
+
+    while len(dog_queue) != 0:
+    # Assign dog to check to current_dog
+        current_dog = dog_queue.popleft()
+
+        if len(kennel_a) == 0:
+            kennel_a.append(current_dog.pop())
+        elif len(kennel_a) > 0 and len(kennel_b) == 0:
+            kennel_b.append(current_dog)
+        else:
+            for dog in kennel_a:
+                if current_dog[0] != dog and current_dog[1] != dog:
+                    kennel_a.append(current_dog)
+                    break
+        # elif len(kennel_b) == 0:
+        #     kennel_b.append(current_dog)
+        if len(kennel_b) > 0:
+            for dog in kennel_b:
+                if current_dog[0] != dog and current_dog[1] != dog:
+                    kennel_b.append(current_dog)
```

Comment:
🤔 I'm not sure I 100% understand what you are trying to do here. 

`current_dog` is a list of all the dogs some dog in the graph (`current_dog[-1]` based on line 18 of your implementation) dislikes. This list of disliked dogs could be longer than two dogs.

I'm not sure why you are comparing them to each dog currently in the kennel, since you don't necessarily know if these dogs dislike each other.